package edu.kh.jdbc.run;

import java.sql.SQLException;

import edu.kh.jdbc.model.service.TestService;
import edu.kh.jdbc.model.vo.TestVO;

public class Run3 {

	public static void main(String[] args) {
		//update
		TestService service = new TestService();
		
		
		TestVO vo2 = new TestVO(2,"제목바꾸기","내용바꾸기");
		
		try {
			try {
				int result2 = service.update(vo2);
			} catch (Exception e) {
				// TODO Auto-generated catch block
				e.printStackTrace();
			}
		} catch (SQLException e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		}
		// 번호, 제목, 내용 입력받아
		
		// 번호가 일치하는 행의 제목, 내용 수정
		
		// 수정 성공시 -> 수정되었습니다.
		// 수정 실패 시 -> 일치하는 번호가 없습니다.
		// 예외 발생 시 -> 수정 중 예외가 발생했습니다.
		
	}

}
